server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -760,6 +769,12 @@ public void run() {
     }
     final AccumuloConfiguration aconf = getConfiguration();
 
+    final long onDemandUnloaderInterval =
+        aconf.getTimeInMillis(Property.TABLE_ONDEMAND_UNLOADER_INTERVAL);
```

Comment:
This is a per table prop but it seems like this is running at the tserver level, so if different tables have different intervals then not sure that would have any impact.

Reply:
Updated in 6409f84 to be a TSERV property